### PR TITLE
[test](mtmv)Test and check the differences between MTMV and OLAP

### PR DESCRIPTION
**Pull request by zfr9527**
Test and check the differences between MTMV and OLAP

### CODE REVIEW

**Comment by zfr9527** [comment]
run buildall

File: regression-test/suites/mtmv_p0/test_mtmv_sql_cache_and_profile.groovy [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        """
+    sql """
+        REFRESH MATERIALIZED VIEW ${mvName} AUTO;
+    """
+
+    def jobName = getJobName(dbName, mvName)
+    waitingMTMVTaskFinished(jobName)
+
+    sql """set enable_sql_cache=true;"""
+    sleep(30 * 1000)
```
Comment (zddr) [inline regression-test/suites/mtmv_p0/test_mtmv_sql_cache_and_profile.groovy:66]: why sleep 30?

Comment (zfr9527) [reply regression-test/suites/mtmv_p0/test_mtmv_sql_cache_and_profile.groovy:66]: It is necessary to wait for a while to make the sql_cache function take effect.

**Comment by zfr9527** [comment]
run buildall

**Review by zddr — APPROVED** [review]

**Comment by zfr9527** [comment]
run buildall

**Review by zddr — APPROVED** [review]

**Review by morrySnow — DISMISSED** [review]

**Review by zddr — APPROVED** [review]

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Review by zddr — APPROVED** [review]

**Comment by zfr9527** [comment]
run buildall

**Comment by morrySnow** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run p0

**Review by zddr — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Pull request merged**